src/wix/WixToolset.Converters/WixConverter.cs

```diff
@@ -2854,6 +2854,45 @@ private static void RenameElementToStandardDirectory(XElement element)
             }
         }
 
+        // If TryGetInnerText succeeds, normalize the string value it returns by replacing all \r's \n's with spaces,
+        // then replacing all multiple spacws with single spaces. This is suitable for use in an XText representing a condition.
+        private static bool TryGetNormalizedInnerText(XElement element, out string value, out List<XNode> comments)
+        {
+            var found = TryGetInnerText(element, out var localValue, out comments);
+
+            if (found && null != localValue)
+
```

Comment:
nit: Unnecessary blank line.

Reply:
This is my style habit. No problem, I'll fix it.

Reply:
I thought that you were referring to line 2862 when I wrote my last comment. Line 2864 is simply an error. That's the line I will remove.